tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -471,13 +486,63 @@ func constructWinRMConfig() (winRMConfig *compute.WinRMConfiguration) {
 	return winRMConfig
 }
 
+// constructAdditionalContent constructs the commands need to executed upon windows node set up.
+// "AutoLogon" setting is responsible to logging into the windows instance and "FirstLogonCommands" is responsible
+// for executing the commands on start i.e in this case it will set up WinRM for Ansible to execute remote commands.
+func constructAdditionalContent(instanceName, adminUserName, adminPassword string) *[]compute.AdditionalUnattendContent {
+	firstLogonData :=
+		"<FirstLogonCommands> " +
+			"<SynchronousCommand> " +
+			"<CommandLine>cmd /c mkdir \"%TEMP%\"\\script</CommandLine> " +
+			"<Description>Create the script directory</Description> " +
+			"<Order>11</Order> " +
+			"</SynchronousCommand> " +
+			"<SynchronousCommand> " +
+			"<CommandLine>cmd /c copy C:\\AzureData\\CustomData.bin " +
+			"\"%TEMP%\"\\script\\winrm.ps1\"</CommandLine> " +
+			"<Description>Move the CustomData file to the working directory</Description> " +
+			"<Order>12</Order>" +
+			"</SynchronousCommand> " +
+			"<SynchronousCommand> " +
+			"<CommandLine>cmd /c powershell.exe -sta -ExecutionPolicy Unrestricted -file " +
+			"\"%TEMP%\"\\script\\winrm.ps1</CommandLine> " +
+			"<Description>Execute the WinRM enabling script</Description> " +
+			"<Order>13</Order> " +
+			"</SynchronousCommand> " +
+			"</FirstLogonCommands>"
+
+	autoLogonData := fmt.Sprintf("<AutoLogon><Domain>%s</Domain><Username>%s</Username><Password><Value>%s</Value></Password>"+
+		"<LogonCount>1</LogonCount><Enabled>true</Enabled></AutoLogon>", instanceName, adminUserName, adminPassword)
+	additionalContent := &[]compute.AdditionalUnattendContent{
+		{
+			PassName:      "OobeSystem",
```

Comment:
What is this `OobeSystem`?

Reply:
If the azure SDK provides constants for these, please use them.